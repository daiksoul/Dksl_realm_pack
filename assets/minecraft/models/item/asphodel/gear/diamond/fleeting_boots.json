{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/diamond_boots",
		"layer1": "misc/fleeting_symbol"
	}
}
