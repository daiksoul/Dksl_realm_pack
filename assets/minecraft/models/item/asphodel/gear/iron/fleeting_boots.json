{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/iron_boots",
		"layer1": "misc/fleeting_symbol"
	}
}
