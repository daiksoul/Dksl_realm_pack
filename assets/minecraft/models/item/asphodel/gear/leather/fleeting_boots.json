{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/leather_boots",
		"layer1": "item/leather_boots_overlay",
		"layer2": "misc/fleeting_symbol"
	}
}
