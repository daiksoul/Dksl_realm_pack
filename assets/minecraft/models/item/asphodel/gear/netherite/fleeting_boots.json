{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/netherite_boots",
		"layer1": "misc/fleeting_symbol"
	}
}
