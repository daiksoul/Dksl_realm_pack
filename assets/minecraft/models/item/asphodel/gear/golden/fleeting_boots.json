{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/golden_boots",
		"layer1": "misc/fleeting_symbol"
	}
}
